tools/include/eosio/codegen.hpp

```diff
@@ -287,6 +288,26 @@ namespace eosio { namespace cdt {
             }
          }
 
+         // Generate get_sync_call_func_name which returns called function name
+         static void create_get_sync_call_func_name(std::stringstream& ss) {
```

Comment:
This function is used by `sync_call` entry point to find out the called function name https://github.com/AntelopeIO/cdt-lld/blob/dceb6185fdcb982384929e61bf58de68ac5eac08/wasm/Writer.cpp#L1265.

Should we move out the called function name from the payload data and place it as an argument of `call()` host function and `sync_call()` entry point? This will save one host function call to `get_call_data()` (we have another call to `get_call_data()` in the actual function call dispatcher https://github.com/AntelopeIO/cdt/blob/f669a1ec676b0db2a5f88a6fa6d9666d67064a88/tools/include/eosio/codegen.hpp#L238)